core/metacling/src/TCling.cxx

```diff
@@ -1720,6 +1690,45 @@ namespace {
 
 }
 
+////////////////////////////////////////////////////////////////////////////////
+///\returns true if the module map was loaded, false on error or if the map was
+///         already loaded.
+bool TCling::RegisterPrebuiltModulePath(const std::string &FullPath,
+                                        const std::string &ModuleMapName /*= "module.modulemap"*/) const
+{
+   assert(llvm::sys::path::is_absolute(FullPath));
+   Preprocessor &PP = fInterpreter->getCI()->getPreprocessor();
```

Comment:
Could pass `fInterpreter` as argument.

Reply:
See above. We can’t make it file static.